webpack.config.js

```diff
@@ -0,0 +1,43 @@
+const path = require('path');
```

Comment:
Please, add the following changes to the Webpack config:
1. Investigate and add `autoprefixer` (through `postcss`)
2. Extract styles in separate bundle `bundle.css` using appropriate plugins for Webpack 4.
3. Add sourcemaps only in `Development` environment. Environment is configured with `NODE_ENV` variable. Read more about it in Webpack docs.